src/tests/base/test_cross_selling.py

```diff
@@ -0,0 +1,634 @@
+
+        Reduced Ticket    23.00          11.50
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        '''
+    )
+    check_cart_behaviour(
+        event,
+        cart_contents=''' Price     Discounted
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+
+        Reduced Ticket    23.00          11.50
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        '''
+    )
+    check_cart_behaviour(
+        event,
+        cart_contents=''' Price     Discounted
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        Tickets     Reduced Ticket      23.00                11.50            2
+        '''
+    )
+    check_cart_behaviour(
+        event,
+        cart_contents=''' Price     Discounted
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+
+        Reduced Ticket    23.00          11.50
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        Tickets     Reduced Ticket      23.00                11.50            1
+        '''
+    )
+    print("The interesting part:")
+    check_cart_behaviour(
+        event,
+        cart_contents=''' Price     Discounted
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+
+        Reduced Ticket    23.00          11.50
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        Tickets     Reduced Ticket      23.00                11.50            1
+        '''
+    )
+    check_cart_behaviour(
+        event,
+        cart_contents=''' Price     Discounted
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+        Regular Ticket    42.00          42.00
+
+        Reduced Ticket    23.00          11.50
+        ''',
+        recommendations='''             Price     Discounted Price    Max Count
+        Tickets     Reduced Ticket      23.00                11.50            2
+        '''
+    )
+
+
+@scopes_disabled()
+@pytest.mark.django_db
+@pytest.mark.skip("currently unsupported (cannot give discount to specific product on minimum cart value)")
```

Comment:
I think I left this in so far because this was one of our example use cases (the champagne example), and it's not supported now...? So we should at least think about whether we want to support this in one way or another, and remove it otherwise.